src/swaps/blindex/BlindexSwapProvider.js

```diff
@@ -0,0 +1,509 @@
+import BN from 'bignumber.js'
```

Comment:
I see you're using both `bignumber.js` and `ethers.BigNumber`
do we really need both?

Reply:
Liquality uses mostly bignumber.js internally (e.g. data returned from overriden SwapProvider methods return bignumber.js objects). BestRouteService and UniswapSwapProvider, for which I copied some code uses ethers.BigNumber. BestRouteService just for calculations, and we might change it, but code from UniswapSwapProvider uses ether.BigNumber to use _toHexString_ method for _encodeFunctionData_ invocation.

TLDR
I would keep using both.